README.md

```diff
@@ -57,6 +57,8 @@ it needs to be disabled.
 Download the archive matching your host OS from the Minishift [releases page](https://github.com/minishift/minishift/releases) and unpack it. Then copy the contained binary to your preferred
 location and optionally ensure it is added to your _PATH_.
 
+**Note**: For Windows OS, you can run the binary from any location in the drive where your `%USERPROFILE%` directory is located. Mostly this is in the `C:` drive.
```

Comment:
@hferentschik , I don't think that is the recommended way. 

https://answers.microsoft.com/en-us/windows/forum/windows_7-desktop/change-the-default-location-of-userprofile-move/dd5a0720-909e-4b71-b665-ba8af19a104f?auth=1